it/controllers/testSupport/ITCoreTestData.scala

```diff
@@ -51,6 +51,20 @@ trait ITCoreTestData extends TryValues {
   def broughtIntoUkFullAnswers = broughtIntoUkPartialAnswers.success.value
     .set(BroughtIntoUKPage, true)
 
+  def addASmallProducerPartialAnswers = emptyUserAnswers
+    .set(OwnBrandsPage, true).success.value
```

Comment:
just a tip, you can use the already defined def here like  exemptionsForSmallProducersFullAnswers which is same as this one